### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Monitoring_Screen Reader: Narrator is not reading proper description of the graph

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#94]
**Environment Details:**
URL:https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS1.3.1

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Monitoring and press enter
6) Start Screen Reader (Narrator/NVDA).
7) Verify screen reader is reading all the contents, controls, links and images properly throughout the process.

**Actual Result:**
Narrator is not reading proper description of the graph after pressing capslock+rt.arrow in the page 

**Expected Result:**
Screen Reader should read all the contents, controls, links and images properly throughout the process.
Here Narrator should read the tooltip shown on the help (?) and also it should read proper description of the graph including data and the contents shown in the x-axis

**User Impact:** 
Screen Reader users won't be able to understand the contents of the graph if screen reader does not read the contents properly

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachments:**
[Bug_94_Screen Reader_Issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2323461/Bug_94_Screen.Reader_Issue.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/236

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #MAS1.3.1; #C+E-H; #.NETWebsites; #Chrome; #NVDA; #Edge; #Narrator

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by fsteffi** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will be removed.
